amida-tech/cda2r4
# Feature/upgrading2r4

#### What does this PR do?
Creates a Test Deployment Pipeline for C-CDA by:
* Updating support for the current C-CDA standard
* Updating Maven Dependencies
* Updating value-maps
* Updating narrative properties
* Making code changes to fix errors caused by upgrade
* Creating a wrapper that reads a file and outputs a file with FHIR R4
#### Related JIRA tickets:
* [SAR-9](https://jira.amida.com/browse/SAR-9)
#### How should this be manually tested?
Steps:
* checkout branch
* run `mvn clean install`
* run `src/main/java/pipeline_wrapper/wrapper.java` with the arguments `C-CDA_R2-1_CCD.xml`  `C-CDA_R2-1_CCD_OUTPUT.json`
* Make sure to give full path for those files
* open to `/Users/henry/Documents/GitHub/cda2r4/src/test/resources/output/C-CDA_R2-1_CCD_OUTPUT.json`
